apis/r/R/RcppExports.R

```diff
@@ -69,17 +69,16 @@ check_arrow_array_tag <- function(xp) {
 #'   \item{\code{sr_next}}{returns the next chunk}
 #' }
 #'
-#' @param ctx An external pointer to a TileDB Context object
 #' @param uri Character value with URI path to a SOMA data set
+#' @param config Named chracter vector with \sQuote{key} and \sQuote{value} pairs
+#' used as TileDB config parameters. If unset default configuration is used.
```

Comment:
I think we should remove "If unset default configuration is used." as it is a mandatory parameter.

Reply:
And see below -- this is again just the shadow.

Reply:
done